Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -37,7 +37,7 @@ HexEditorWidget::HexEditorWidget()
         m_statusbar->set_text(1, String::formatted("Edit Mode: {}", edit_mode == HexEditor::EditMode::Hex ? "Hex" : "Text"));
         m_statusbar->set_text(2, String::formatted("Selection Start: {}", selection_start));
         m_statusbar->set_text(3, String::formatted("Selection End: {}", selection_end));
-        m_statusbar->set_text(4, String::formatted("Selected Bytes: {}", abs(selection_end - selection_start) + 1));
+        m_statusbar->set_text(4, String::formatted("Selected Bytes: {}", abs(selection_end - selection_start)));
```

Comment:
This doesn't seem to fit in with the other changes, it seems to be a reasonable fix though
just checking if it's supposed to be in this commit

Reply:
It is intentional. It's a ninja patch to a dumb issue, tangentially related to the topic of the PR (content selection). The selection count is off by one.

Here's another dumb issue that I didn't fix in this PR. I'll ninja patch it in one of the next.

```diff
diff --git a/Userland/Applications/HexEditor/FindDialog.cpp b/Userland/Applications/HexEditor/FindDialog.cpp
index e5ec83f42..870ed5365 100644
--- a/Userland/Applications/HexEditor/FindDialog.cpp
+++ b/Userland/Applications/HexEditor/FindDialog.cpp
@@ -26,7 +26,7 @@ struct Option {
 };
 
 static const Vector<Option> options = {
-    { "ACII String", OPTION_ASCII_STRING, true, true },
+    { "ASCII String", OPTION_ASCII_STRING, true, true },
     { "Hex value", OPTION_HEX_VALUE, true, false },
 };
```

Reply:
Alright, looks okay then : )